ci/send-snapshots/jobs/production-nifi-heart-beat.yml

```diff
@@ -0,0 +1,42 @@
+resources:
+  - name: time-interval
```

Comment:
I'd name it something like `every-60-seconds` so you can see on the UI what it does